modules/common/time/time.h

```diff
@@ -147,6 +147,14 @@ class Clock {
 
 inline Clock::Clock() {
   mode_ = FLAGS_use_cyber_time ? ClockMode::CYBER : ClockMode::SYSTEM;
+  // Force to use Cyber time is USE_SIM_TIME is set
+  const char* use_sim_time = ::getenv("USE_SIM_TIME");
```

Comment:
Please use `apollo::cyber::common::GetEnv` from `cyber/common/environment.h` be to consistent.